ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -1868,7 +2093,9 @@ ReadOverlayArray2:
     ; A is already 16 bit here.
     REP #$10 ; Set X and Y in 16bit mode.
 
-    JSL.l ReadOverlayArray : TAY
+    ; ReadOverlayArray
```

Comment:
Yeah same as before just curious why this is preferred over jumping.

Reply:
Same as before, just a slightly different use case that I didn't bother to make a ReadOverlayArray3 function for.